src/delphi/dataset/mock_per_token_performance.py

```diff
@@ -0,0 +1,47 @@
+from math import log2
+from random import random, seed
```

Comment:
unless importing from our own library, default to importing modules instead of specific functions / classes
```suggestion
import math
import random
```